File: Assignment_031/Program2.java
import java.util.Scanner;

class Bitwise
{
    public int OFFBit(int iNo)
    {
        int iMask = 576;  
        int iResult = 0;

        iResult = iNo ^ iMask;

        return iResult;
    }

}
class Program2
{
    public static void main(String A[])
    {
        int iValue = 0;
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter the Number : ");
        iValue = sobj.nextInt();

        Bitwise bobj = new Bitwise();

        int iRet = 0;

        iRet = bobj.OFFBit(iValue);

        System.out.println("Updated Number is : "+iRet);

        sobj = null;
        bobj = null;


    }
}

File: Assignment_043/program2.c
#include<stdio.h>

int Maximum(int iNo)
{
    static int iDigit = 0;
    static int iMax = 0;

    if(iNo != 0)
    {
        iDigit = iNo % 10;
        
        if(iMax < iDigit)
        {
            iMax = iDigit;
        }

        iNo = iNo / 10;

        Maximum(iNo);
    }

    return iMax;
}
int main()
{
    int iValue = 0;
    int iRet = 0;

    printf("Enter a number : \n");
    scanf("%d",&iValue);

    iRet = Maximum(iValue);

    printf("%d\n",iRet);


    return 0;
}